File: UIBS_Enquiry_Module/src/main/java/com/uibs/enquiry/service/EnquiryService.java
package com.uibs.enquiry.service;

import java.util.List;

import com.uibs.enquiry.model.Email;
import com.uibs.enquiry.model.Enquiry;

public interface EnquiryService {

	public Enquiry saveEnquiry(Enquiry enquiry);

	public void sendEnquiryMail(Enquiry enquiry, Email email);

	public List<Enquiry> getEnquiries();

	public Enquiry getEnquiry(long id);

}

File: uibs_home_loan/src/app/services/enquiry.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Enquiry } from '../model/enquiry';

@Injectable({
  providedIn: 'root'
})
export class EnquiryService {

  constructor(private http:HttpClient) { }

  saveEnquiry(enquiry:Enquiry) {
    return this.http.post("http://localhost:1111/uibs/saveEnquiry", enquiry);
  }
}
